app/src/main/java/com/mathroule/demoapp/MainActivity.java

```diff
@@ -28,6 +31,30 @@ public void onClick(View view) {
                         .setAction("Action", null).show();
             }
         });
+
+        try {
+            String text = Helper.getRandomButtonLabel();
+            setButonLable(text);
+
+            if (text == "something") {
+                setButonLable("something");
+            } else           if (text == "something-else") {
+                setButonLable("something-else");
+            }else           if (text == "something-else") {
+                setButonLable(null);
+            }
+        } catch (Exception e) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/a45dcc50006f21ef726783b1160a50e6497aa0e6/app/src/main/java/com/mathroule/demoapp/MainActivity.java#L46"></span>Exception is caught when Exception is not thrown in com.mathroule.demoapp.MainActivity.onCreate(Bundle)</td>
    </tr>
  </tbody>
</table>